src/main/java/com/direwolf20/buildinggadgets/common/building/PlacementTarget.java

```diff
@@ -0,0 +1,37 @@
+package com.direwolf20.buildinggadgets.common.building;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+/**
+ * Indicates the position and type of block to use for placing something in a {@link World}.
+ */
+public final class PlacementTarget {
```

Comment:
I'm not seeing a use for this class? Can you confirm if this is correct or not. If it's being used then ignore.

Reply:
It is not right now. However I think @MajorTuvok requested me to keep it (not sure if he still wanted to right now, because there is one already in `1.13.x`).

Reply:
I needed that class so desperatly, that I added it there...
If you want you can remove it...